rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java

```diff
@@ -29,5 +31,17 @@ public JsonObject map(Claim claim) {
             .add("defendantEmailAddress", claim.getDefendantEmail())
             .build();
     }
+
+    private String mapCountyCourtJudgementType(CountyCourtJudgmentType ccjType) {
```

Comment:
Typo in "Judgement" here and elsewhere in new code. (Yeah, I know, it crops up with both spellings throughout the code, but let's keep new code correct when we can!)

Reply:
Fixed